### DISCUSSION TITLE
Downloading pytorch_model.bin

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#1938] [Q&A]
When I swapped the model loaded for another one it started with this in cmd.

Is there a fix? 

![image](https://github.com/vladmandic/automatic/assets/117453319/c3ca3fc5-b518-4ebf-84fe-4dcb78c6924f)

**Comment by vladmandic** [comment]
There is no fix because normally there is no issue.
Check if you have safetensors enabled in as allowed format in settings - diffusers.

Or just delete config.json so configuration is reset, defaults should just work.
